### ISSUE TITLE
"Copy" button is not accessible with any keyboard key: A11y_VisualStudioCodeServices_Docs_RemoteDevlopmentTipsAndTricks_Keyboard

### ISSUE THREAD

**Issue by shindepra** [microsoft/vscode-docs#8431]
**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### GitHub Tags
#A11ySev2; #A11yTCS; #WCAG2.1.1; #Chrome; #DesktopWeb; #Win11; #Visual Studio Code Services; #BM_VisualStudioCodeServices_Web_May2025; #keyboard; #A11yAuto; #TabStops;

### Environment Details
Application Name: Visual Studio Code Services
URL: https://code.visualstudio.com/docs/remote/troubleshooting
Microsoft Windows 11 Enterprise (24H2) Build on 26100.4061
Chrome Version 136.0.7103.114 (Official Build) (64-bit)

### Repro Steps

1. Launch the application URL: https://code.visualstudio.com/docs/remote/troubleshooting
2. Tab till "Edit" button and press enter.
3. Observe that "Copy" button is not accessible with any keyboard key.

### Actual Result
"Copy" button is not accessible with any keyboard key.

### Expected Result
Copy button should be accessible with TAB key.

### User Impact
Keyboard user will not be able to access the copy button and will not be able to copy the data using any keyboard key

### Attachment

https://github.com/user-attachments/assets/77d747ce-1a45-4a77-9038-e4cef86bcef9

**TabStops Attachment**

<img width="957" alt="Image" src="https://github.com/user-attachments/assets/9f811469-aa0c-4055-8e09-e780eb923050" />

**Comment by ntrogh** [comment]
@shindepra This issue should be resolved now.
